plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -25,30 +25,36 @@
 #include "resource.h"
 #endif
 
-const char* dialog_message(int ID, bool* updateText) {
+#ifdef _WIN32
+#define _cvtString( s ) L##s
```

Comment:
Leading underscores shouldn't be used. They're reserved for use by the standard for implementation use.